src/chrome/content/rules/Ghostery_Enterprise.com.xml

```diff
@@ -1,37 +1,28 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://ghosteryenterprise.com/ => https://ghosteryenterprise.com/: (51, "SSL: no alternative certificate subject name matches target host name 'ghosteryenterprise.com'")
-Fetch error: http://site-cdn1.ghosteryenterprise.com/ => https://site-cdn1.ghosteryenterprise.com/: (60, 'SSL certificate problem: certificate has expired')
-Fetch error: http://site-cdn2.ghosteryenterprise.com/ => https://site-cdn2.ghosteryenterprise.com/: (60, 'SSL certificate problem: certificate has expired')
-Fetch error: http://site-cdn3.ghosteryenterprise.com/ => https://site-cdn3.ghosteryenterprise.com/: (60, 'SSL certificate problem: certificate has expired')
-Fetch error: http://site-cdn4.ghosteryenterprise.com/ => https://site-cdn4.ghosteryenterprise.com/: (60, 'SSL certificate problem: certificate has expired')
-Fetch error: http://site-cdn5.ghosteryenterprise.com/ => https://site-cdn5.ghosteryenterprise.com/: (60, 'SSL certificate problem: certificate has expired')
-Fetch error: http://staging.ghosteryenterprise.com/ => https://staging.ghosteryenterprise.com/: (28, 'Connection timed out after 20000 milliseconds')
-Fetch error: http://www.ghosteryenterprise.com/ => https://www.ghosteryenterprise.com/: (51, "SSL: no alternative certificate subject name matches target host name 'www.ghosteryenterprise.com'")
-
 	For other Evidon coverage, see Evidon.xml.
 
+	Non-functional hosts
+		Timeout was reached:
+			 - choices.ghosteryenterprise.com
+
+		SSL peer certificate was not OK:
+			 - ghosteryenterprise.com
+			 - www.ghosteryenterprise.com
+			 - livescan.ghosteryenterprise.com
+
+		Peer certificate cannot be authenticated with given CA certificates:
+			 - cdn.ghosteryenterprise.com
+			 - help.ghosteryenterprise.com
+			 - my-cdn.ghosteryenterprise.com
+			 - site-cdn1.ghosteryenterprise.com
+			 - site-cdn2.ghosteryenterprise.com
```

Comment:
I don't prefer regex because I am used to extract the commented domain in the following way. Thanks anyway!!

```bash
grep ' - ' Ruleset.xml | sed 's/\s*- \(.*\).*/\1/g'
```